app/views/categories/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<section class="categories_sect">
+<div class="navigation">
+  <span class="hamburger">&#9776;</span>
+   <h1>Categories</h1>
+  <%= link_to "Sign Out", destroy_user_session_path, method: :delete, class: "sign-out-butt" %>
+</div>   
+    <div class="cat_page">
+  <div id="categories">
+    <% if @categories.length == 0 %>
+      <p class="no-categories"> There are no categories yet,,,,!create one</p>
+    <% else %>
+      <% @categories.each do |category| %>
+      <div class="cat_name">
+         <%= link_to category_records_path(category) do %>
+         <%= render category %>
+        <% end %>
+      </div>
+      <% end %>
+    <% end %>
+  </div>
+  <div class="cat_link_cont">
+  <div class= "inner">
+    <%= link_to "New category", new_category_path, class:"cat_link" %>
```

Comment:
- Kindly follow the design guidelines for the add new category button:
   - [ ] Please change the background color to match the design.
   - [ ] Kindly increase the inner white space.

| Your design | Guidelines |
| --- | --- |
| ![image](https://github.com/Christelle-12/Budget-manager/assets/24703171/f17a8cd6-bd2b-40e4-88cf-06c10ce3db37) | ![image](https://github.com/Christelle-12/Budget-manager/assets/24703171/1fcbe411-668b-4b94-8f3f-fdffb26da41e) |